cmd/mycelo/main.go

```diff
@@ -426,11 +444,12 @@ func loadBot(ctx *cli.Context) error {
 
 	// TODO: Pull all of these values from env.json
 	return loadbot.Start(runCtx, &loadbot.LoadBotConfig{
-		Accounts:         env.DeveloperAccounts(),
-		Amount:           big.NewInt(10000000),
-		TransactionDelay: 1 * time.Second,
+		Accounts:              env.DeveloperAccounts(),
+		Amount:                big.NewInt(10000000),
+		TransactionsPerSecond: env.Config.LoadTestTPS,
+		ClientCount:           1,
 		ClientFactory: func() (*ethclient.Client, error) {
-			return ethclient.Dial("http://localhost:8545")
+			return ethclient.Dial(env.IPC())
```

Comment:
i used http since i didn't know the IPC support multiple connections or how well handles that

Reply:
The answer is that IPC is a lot better than HTTP. HTTP works well up to maybe 10 or 20 TPS and then just falls over. I've also done work to limit the number of clients used (I had to tune that with HTTP, but one IPC client supports as many TPS as needed).